packages/directlinespeech/src/SpeechSynthesisAudioStreamUtterance.js

```diff
@@ -0,0 +1,20 @@
+import EventTarget, { defineEventAttribute } from './external/event-target-shim';
+
+class SpeechSynthesisAudioStreamUtterance extends EventTarget {}
+
+defineEventAttribute(SpeechSynthesisAudioStreamUtterance, 'boundary');
+defineEventAttribute(SpeechSynthesisAudioStreamUtterance, 'end');
+defineEventAttribute(SpeechSynthesisAudioStreamUtterance, 'error');
+defineEventAttribute(SpeechSynthesisAudioStreamUtterance, 'start');
+
+function fromAudioStream(audioStream) {
+  const utterance = new SpeechSynthesisAudioStreamUtterance();
+
+  utterance.audioStream = audioStream;
```

Comment:
Is this the convention?

Alternative:

```javascript
class SpeechSynthesisAudioStreamUtterance extends EventTarget {
  constructor (audioStream) {
    super();
    this.audioStream = audioStream;
  }
}
```

Reply:
The original Web Speech API is like this. It accept one `string` argument.

```js
new SpeechSynthesisUtterance(text);
```

Originally, want to keep the API not breaking *if* someone steal our `SpeechSynthesisUtterance` out and passed a string to it.

I am modifying it like what you said, and do argument validation.